docs/src/main/paradox/docs/delta/api/current/views/sparql-view-api.md

```diff
@@ -116,7 +304,7 @@ where...
 - `lastProcessedEventDateTime` - timestamp of the last event processed by the view
 - `delayInSeconds` - number of seconds between the last processed event timestamp and the last known event timestamp
  
-### Restart view
+## Restart indexing
```

Comment:
If you want to call it `Restart indexing` here (which I agree), then let's also call it `Restart indexing` in elasticsearch (currently is called `restart view`)